python/test/test_api.py

```diff
@@ -70,7 +70,7 @@
     exit_on_error=True,
     strict_model_config=False,
     model_control_mode=tritonserver.ModelControlMode.EXPLICIT,
-    exit_timeout=10,
+    exit_timeout=30,
```

Comment:
one small thing: if we want to match default behavior - we can remove setting this value entirely - and rely on default.

Reply:
Makes sense. I'd like to expand on the testing a bit when I get more cycles, so can consider just removing this later on. Will keep it explicit for now to make sure its clear that its 30 seconds and is configurable if need be to new readers.